src/pages/resources.js

```diff
@@ -3,11 +3,42 @@ import React from "react";
 import Layout from "components/layout";
 import { Newsletter } from "components";
 import Container from "components/container";
+import ArrowIcon from "assets/images/svgs/arrow.svg";
 import ctl from "@netlify/classnames-template-literals";
 
 import { ResourceCategory, ResourcesHeader } from "templates/resources";
 import { graphql, useStaticQuery } from "gatsby";
 
+const arrowsContainerStyle = ctl(`
```

Comment:
I'd suggest we move the styles below the component itself, should in case the styles get long in the future.